test/integration/targets/nxos_bgp/tests/cli/sanity.yaml

```diff
@@ -0,0 +1,135 @@
+---
+- debug: msg="START TRANSPORT:CLI nxos_bgp sanity test"
+
+- set_fact: log_neighbor_changes="true"
```

Comment:
Ok fixed. If there are more issues with titanium specifically can we get a whole rundown on the issue rather than piece meal info ? This is incredibly frustating. Ideally, a test against an irrelevant platform shouldn't gate commits.